src/sentry/api/serializers/models/discoversavedquery.py

```diff
@@ -32,7 +33,9 @@ def serialize(self, obj, attrs, user, **kwargs):
             "version": obj.version or obj.query.get("version", 1),
             "dateCreated": obj.date_created,
             "dateUpdated": obj.date_updated,
-            "createdBy": six.text_type(obj.created_by_id) if obj.created_by_id else None,
+            "createdBy": serialize(obj.created_by, serializer=UserSerializer())
+            if obj.created_by
+            else None,
```

Comment:
We should check that this doesn't break discover1. While its not long for the world we shouldn't break it before we remove it.

Reply:
:+1: Clicked around v1 and it looks good. Grepping through the static code createdBy is included in a few spots but not used AFAICT.